### PR TITLE
[planetstation] mushroom people act 2

### PR DESCRIPTION
**Pull request by Militaires**
just the race for now

### CODE REVIEW

**Comment by Militaires** [comment]
why is it conflicting aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by FantasticFwoosh** [comment]
Bman's bizarre adventure Act 2: PR closure tendency

> stick a this one man, it'll see the light of day.

**Comment by Militaires** [comment]
i thought learning to code for tgstation would make me more enthusiastic but instead it made me a tired man

longing to die.

**Comment by jughu** [comment]
Nanotrasen % Knuckles
Mushroom zone Act 2
Start

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -160,4 +160,4 @@ GLOBAL_LIST_INIT(numbers_as_words, list("One", "Two", "Three", "Four",
 		L += "\Roman[i]"
 	return L
 
-GLOBAL_LIST_INIT(station_numerals, greek_letters + phonetic_alphabet + numbers_as_words + generate_number_strings())
+var/global/list/station_numerals = greek_letters + phonetic_alphabet + numbers_as_words + generate_number_strings()
```
Comment (ChangelingRain) [inline code/_globalvars/lists/flavor_misc.dm]: in one fell swoop, you fucked up

Comment (Militaires) [inline code/_globalvars/lists/flavor_misc.dm]: it's just an unintended revert dud

**Comment by Militaires** [comment]
wait

**Pull request closed without merging**